Dockerfile

```diff
@@ -26,10 +26,10 @@ RUN cat /etc/os-release \
         /tmp/rpms/$([ -d /tmp/rpms/$(uname -m) ] && echo $(uname -m)/)openshift-hyperkube-*.rpm \
     && rpm-ostree cliwrap install-to-root / \
     && rpm-ostree override replace \
-    https://kojipkgs.fedoraproject.org//packages/kernel/6.5.4/300.fc39/x86_64/kernel-6.5.4-300.fc39.x86_64.rpm \
-    https://kojipkgs.fedoraproject.org//packages/kernel/6.5.4/300.fc39/x86_64/kernel-core-6.5.4-300.fc39.x86_64.rpm \
-    https://kojipkgs.fedoraproject.org//packages/kernel/6.5.4/300.fc39/x86_64/kernel-modules-6.5.4-300.fc39.x86_64.rpm \
-    https://kojipkgs.fedoraproject.org//packages/kernel/6.5.4/300.fc39/x86_64/kernel-modules-core-6.5.4-300.fc39.x86_64.rpm \
+    https://kojipkgs.fedoraproject.org//packages/kernel/6.5.4/200.fc38/x86_64/kernel-6.5.4-200.fc38.x86_64.rpm \
```

Comment:
```suggestion
    https://kojipkgs.fedoraproject.org//packages/kernel/6.5.5/200.fc38/x86_64/kernel-6.5.5-200.fc38.x86_64.rpm \
```

Reply:
We want 6.5.4 to match the first 4.14 stable